proprietary/tokens/src/components/ams/table-of-contents.tokens.json

```
@@ -0,0 +1,56 @@
+{
+  "ams": {
+    "table-of-contents": {
+      "color": { "value": "{ams.color.primary-black}" },
+      "font-family": { "value": "{ams.text.font-family}" },
+      "font-size": { "value": "{ams.text.level.5.font-size}" },
+      "font-weight": { "value": "{ams.text.font-weight.normal}" },
+      "gap": { "value": "{ams.space.inside.lg}" },
+      "line-height": { "value": "{ams.text.level.5.line-height}" },
+      "link": {
+        "color": { "value": "{ams.link-appearance.color}" },
+        "outline-offset": { "value": "{ams.focus.outline-offset}" },
+        "text-decoration-line": { "value": "{ams.link-appearance.subtle.text-decoration-line}" },
+        "text-decoration-thickness": { "value": "{ams.link-appearance.text-decoration-thickness}" },
+        "text-underline-offset": { "value": "{ams.link-appearance.text-underline-offset}" },
+        "hover": {
+          "color": { "value": "{ams.link-appearance.hover.color}" },
+          "text-decoration-line": { "value": "{ams.link-appearance.subtle.hover.text-decoration-line}" }
+        },
+        "on-background-dark": {
+          "color": { "value": "{ams.link-appearance.on-background-dark.color}" },
+          "hover": {
+            "color": { "value": "{ams.link-appearance.on-background-dark.color}" }
+          }
+        },
+        "on-background-light": {
+          "color": { "value": "{ams.link-appearance.on-background-light.color}" },
+          "hover": {
+            "color": { "value": "{ams.link-appearance.on-background-light.color}" }
+          }
+        }
+      },
+      "list": {
+        "gap": { "value": "{ams.space.inside.lg}" },
```

This one should be 16 as well.

```suggestion
        "gap": { "value": "{ams.space.inside.md}" },
```